apps/SageAccountWeb/src/style/theme.ts

```diff
@@ -1,9 +1,12 @@
-import { createTheme, Theme } from '@mui/material'
-import { mergeTheme } from 'synapse-react-client/dist/utils/theme'
+import { ThemeOptions } from '@mui/material'
 
 export const latoFont = ['Lato', 'Roboto', 'Helvetica', 'Arial'].join(',')
 
-export const themeOptions = mergeTheme({
+// Merge the default theme (defined in synapse-react-client) with the SageAccountWeb overrides defined here.
+export const sageAccountWebThemeOverrides: ThemeOptions = {
+  styledBackground:
+    "linear-gradient(rgba(255,255,255,0.7), rgba(255,255,255,0.7)), url('https://s3.amazonaws.com/static.synapse.org/images/SynapseLoginPageBackground.svg')",
```

Comment:
StyledOuterContainer now uses this string for the background. This lets us change it between the default (SynapseWebClient) style which is blue, and this version, which applies a white overlay.